### PR TITLE
Target platform and tweaks to prebuildify arguments

### PR DESCRIPTION
**Pull request by ralphtheninja**
Basing this on top of the `--args="-t 8.14.0 --napi --strip"` code

### CODE REVIEW

**Comment by ralphtheninja** [comment]
cc @vweevers

File: build [hunk 1]
```diff
@@ -73,6 +73,10 @@ while true; do
             PLATFORM="$2"
             shift 2
             ;;
+        --args)
+            ARGS="$2"
```
Comment (vweevers) [inline build]: Could we instead use rest arguments (everything after `--`)? E.g.:

`prebuildify-cross --platform=linux --arch=arm --arm-version=7 -- -t 8.14.0 --napi --strip`

File: package.json [hunk 2]
```diff
@@ -20,5 +17,8 @@
   "bugs": {
     "url": "https://github.com/ahdinosaur/prebuildify-cross/issues"
   },
-  "homepage": "https://github.com/ahdinosaur/prebuildify-cross#readme"
+  "homepage": "https://github.com/ahdinosaur/prebuildify-cross#readme",
+  "peerDependencies": {
+    "prebuildify": "*"
```
Comment (vweevers) [inline package.json]: Is this temporary or permanent? I hate peerDependencies :)

Comment (ralphtheninja) [reply package.json]: Wut. Where the hell did this come from?

Comment (ralphtheninja) [reply package.json]: Oh nevermind, I thought this was in `leveldown` repo lol. Yes, I don't like it either.

File: build (hunk 1 above)
Comment (ralphtheninja) [reply build]: I don't mind at all.

File: build [hunk 3]
```diff
@@ -88,7 +88,7 @@ done
 if
   [ "${HELP}" == "y" ] || \
   [ "${ARCH}" == "" ] || \
-  !( [ "${ARCH}" != "x32" ] && [ "${ARCH}" != "x64" ] && [ "${ARCH}" != "arm" ] && [ "${ARCH}" != "arm64" ] ) || \
+  ( [ "${ARCH}" != "x32" ] && [ "${ARCH}" != "x64" ] && [ "${ARCH}" != "arm" ] && [ "${ARCH}" != "arm64" ] ) || \
```
Comment (vweevers) [inline build]: 👍

**Pull request merged**